graylog2-server/src/main/java/org/graylog2/indexer/searches/Searches.java

```diff
@@ -642,6 +641,11 @@ public HistogramResult fieldHistogram(String query,
                 .filter(Objects::nonNull)
                 .collect(Collectors.toList());
 
+            final Optional<String> nonNumericFieldError = errors.stream().filter(error -> error.startsWith("Expected numeric type on field")).findAny();
+            if (nonNumericFieldError.isPresent()) {
```

Comment:
Would it make sense to store all errors for this operation in the `errorDetails` field of the exception?